holidayextras/ui-toolkit
# Alert component - Don't render a empty h4 tag if a title prop is not passed in

#### What does this PR do? (please provide any background)

This changes the alert component so that if you don't pass in a title property then it will not render a blank h4. I have also updated the README so it details the correct command for building the toolkit

#### What tests does this PR have?

New unit tests to cover both scenarios

#### How can this be tested?

run the docs and delete the title property from one of the example and inspect the element there should not be a h4

#### Screenshots / Screencast
#### What gif best describes how you feel about this work?

![](https://media.giphy.com/media/FFb9yZK6t0oDu/giphy.gif)

---
#### Developer Definition of Done/Quality Checklist (for PR author to complete BEFORE code review):

#### Software Engineer or Developer review:

#### Software Engineer or project guru review:
